policy/modules/kernel/filesystem.if

```diff
@@ -2324,6 +2342,76 @@ interface(`fs_mounton_fusefs',`
 	allow $1 fusefs_t:dir mounton;
 ')
 
+########################################
+## <summary>
+##	Mounton FUSEFS files.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`fs_mounton_fusefs_files',`
+	gen_require(`
+		type fusefs_t;
+	')
+
+	allow $1 fusefs_t:file mounton;
+')
+
+########################################
+## <summary>
+##	Allow FUSEFS files to be an entrypoint
+##	for the specified domain.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+interface(`fs_fusefs_entrypoint',`
+	gen_require(`
+		type fusefs_t;
+	')
+
+	allow $1 fusefs_t:file entrypoint;
```

Comment:
This is horrifying.

Reply:
Normally rootless podman will do `setexeccon()` to transition to `container_user_t` when executing a FUSEFS file, if FUSE is being used, so `fusefs_t` must be an entrypoint for `container_t`.

I think what I should do here is rename this interface to `fs_fusefs_entry_type()` and have it call `domain_entry_file($1, fusefs_t)`, but I want to make sure that's the change you want.

Reply:
Yes, that's better.